igc.h

```diff
@@ -25,23 +28,61 @@
 #ifndef IGC_H_INCLUDED_
 #define IGC_H_INCLUDED_
 
-#include <optional>              // for optional
-
 #include <QByteArray>            // for QByteArray
 #include <QDateTime>             // for QDateTime
 #include <QList>                 // for QList<>::const_iterator
 #include <QString>               // for QString, operator+, QStringLiteral
 #include <QVector>               // for QVector
+#include <QHash>                 // for QHash
 
 #include "defs.h"
 #include "format.h"
 #include "gbfile.h"              // for gbfprintf, gbfclose, gbfopen, gbfputs, gbfgetstr, gbfile
+#include <optional>              // for optional
```

Comment:
optional isn't one of ours, it's from the standard library.  The brackets vs. double quotes are a hint.  We like to group the standard libarary includes as a group before the Qt includes.  For example in kml.h <tuple> is included before the Qt includes.